pom.xml

```diff
@@ -54,6 +54,7 @@
     <gitRepositoryName>sonar-scanner-jenkins</gitRepositoryName>
     <version.artifactory.plugin>2.6.1</version.artifactory.plugin>
     <jenkins.version>2.222.4</jenkins.version>
+    <version.cyclonedx.plugin>2.5.3</version.cyclonedx.plugin>
```

Comment:
I'm guessing we're using this same property everywhere? If not, can we align it with the previous one and call it `cyclonedx.plugin.version` for consistency? Otherwise, leave it like this 👍 .

Reply:
Indeed this was factored in the parent pom

Reply:
scratch that, this scanner does not use the parent pom so we have to re-declare everything for this one